includes/import-products.php

```diff
@@ -101,6 +100,11 @@ function dw_add_woocommerce_product($course) {
 			'is_visible'  => true,
 			'is_taxonomy' => true,
 		],
+		'pa_pretty_date' => [
+			'name' => 'pa_pretty_date',
+			'is_visible' => true,
+			'is_taxonomy' => true,
```

Comment:
@Dirkhuethorst, why isn't there a `'value' => $startDate->format('some pretty format')`?

Reply:
We add the value with wp_set_object_terms later on for these, since we no not now them at this point. Somehow without adding `pa_pretty_date` and `pa_slot_time` here they do not get added to the products.